src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx

```diff
@@ -219,4 +219,7 @@ Valid formats are:
   <data name="STANotImplemented" xml:space="preserve">
     <value>Parameter -STA is not supported on this platform.</value>
   </data>
+  <data name="NullElementInArgs" xml:space="preserve">
+    <value>The specified arguments should not contain null elements.</value>
```

Comment:
```suggestion
    <value>The specified arguments must not contain null elements.</value>
```

Reply:
It might also be more helpful to give the index of the argument

Reply:
We throw the exception when run into the first null element, there may be more null elements, so the index may not be that useful.

Reply:
Well with no index we have to guess, with one we at least know where to start